### ISSUE TITLE
I don't know exactly how to use this tool

### ISSUE THREAD

**Issue by rushi3691** [kallqvist/skillshare-downloader#41]
![Annotation 2020-09-29 121133](https://user-images.githubusercontent.com/71341783/94522311-13089900-024d-11eb-9e6e-c119e19c053d.png)
Pls can someone tell me how to fix this problem
and please modify the readme file for better understanding for newbies like me.

**Comment by username-011** [comment]
Execute **pip install python-slugify** in console.
Follow instructions to get cookie, add it to example.py.

For more convenient way than editing example.py every time, you can comment line 10 (add # at the start) and add this at the end:
```
while True:
    dl.download_course_by_class_id((input("Course id: "))
```
ID is a 9 digit number at the end of URL of course. You can paste it to the console with right mouse click.

**Comment by rushi3691** [comment]
yes it works! How can I download subtitles also.

**Comment by rushi3691** [comment]
> Execute **pip install python-slugify** in console.
> Follow instructions to get cookie, add it to example.py.
> 
> For more convenient way than editing example.py every time, you can comment line 10 (add # at the start) and add this at the end:
> 
> ```
> while True:
>     dl.download_course_by_class_id((input("Course id: "))
> ```
> 
> ID is a 9 digit number at the end of URL of course. You can paste it to the console with right mouse click.

yeah . Thanks. it works! Can you pls tell me how to download subtitles also?

**Comment by rushi3691** [comment]
> Execute **pip install python-slugify** in console.
> Follow instructions to get cookie, add it to example.py.
> 
> For more convenient way than editing example.py every time, you can comment line 10 (add # at the start) and add this at the end:
> 
> ```
> while True:
>     dl.download_course_by_class_id((input("Course id: "))
> ```
> 
> ID is a 9 digit number at the end of URL of course. You can paste it to the console with right mouse click.

I'm getting this error while using while loop command:
![err1](https://user-images.githubusercontent.com/71341783/94664615-b5497f00-0328-11eb-8de3-a09c7bb42c26.png)
![error](https://user-images.githubusercontent.com/71341783/94664634-bf6b7d80-0328-11eb-985b-17fcd483fa5d.png)

**Comment by Aman010101** [comment]
Guys please help me. I am just trying to download my class. I dont know anything about coding. I spent an hour to find out I need python. Please help and tell me how exactly I need to the thing to download my steps.
